src/java/org/apache/cassandra/config/GuardrailsOptions.java

```diff
@@ -653,6 +691,25 @@ private static void validateWarnLowerThanFail(long warn, long fail, String name)
                                                       "than the fail threshold %d", warn, name, fail));
     }
 
+    private static void validateWarnGreaterThanFail(long warn, long fail, String name)
+    {
+        if (warn == -1|| fail == -1)
```

Comment:
Nit: missed white space:
```suggestion
        if (warn == -1 || fail == -1)
```